packages/preset-mini/build.config.ts

```diff
@@ -13,5 +13,9 @@ export default defineBuildConfig({
   declaration: true,
   rollup: {
     emitCJS: true,
+    dts: {
+      respectExternal: false,
+    },
   },
+  failOnWarn: false,
```

Comment:
Any reason these are added? `failOnWarn: false` feels dangerous

Reply:
I also feel is not good, but ci error. see https://github.com/unocss/unocss/actions/runs/6608185990/job/17946548533?pr=3149#step:9:277